docs/admin/common_tasks.md

```diff
@@ -0,0 +1,105 @@
+# DAOS Common Tasks
+
+This section describes some of the common tasks handled by admins at a high level. See [System Deployment](./deployment#system-deployment), [DAOS System Administration](./administration.md#daos-system-administration), and [Pool Operations](./pool_operations#pool-operations) for more detailed explanations about each step.
```

Comment:
Is `.md` needed? When clicking through GitHub, links give 404
```suggestion
This section describes some of the common tasks handled by admins at a high level. See [System Deployment](./deployment.md#system-deployment), [DAOS System Administration](./administration.md#daos-system-administration), and [Pool Operations](./pool_operations.md#pool-operations) for more detailed explanations about each step.
```

Reply:
Yes. Thanks for checking.